unit/Makefile

```diff
@@ -31,52 +24,33 @@ LIBS = ../src/ansi-c/ansi-c$(LIBEXT) \
        ../src/assembler/assembler$(LIBEXT) \
        ../src/analyses/analyses$(LIBEXT) \
        ../src/solvers/solvers$(LIBEXT) \
+       # Empty last line
 
-CLEANFILES = $(SRC:.cpp=$(EXEEXT))
+TESTS = catch_entry_point$(EXEEXT) \
```

Comment:
All the Catch tests get compiled into one executable, so the only way to remove them will be to modify the source of the test or remove them from the SRC list above I believe.